src/content/docs/ci-insights/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: CI Insights
+description: Optimize your CI run time, catch flaky tests, and give developers actionable insights so your team ships code faster.
+---
+
+import { Image } from "astro:assets"
+import MainScreenshot from "../../images/ci-insights/main.png"
+import TokenScreenshot from "../../images/ci-insights/token.png"
+import GHSecretsScreenshot from "../../images/ci-insights/gh-secrets.png"
+import {
+  SiPytest,
+  SiGithub,
+  SiCypress,
+  SiVitest,
+} from 'react-icons/si';
+import Docset from '../../../components/DocsetGrid/Docset.astro';
+import DocsetGrid from '../../../components/DocsetGrid/DocsetGrid.astro';
+
+CI Insights helps you monitor your pipelines, detect flaky tests, and optimize
+your CI efficiency. This guide walks you through enabling CI Insights via
+GitHub and covers basic configuration steps.
+
+:::note
+  If you do not see CI Insights in your dashboard, this means you are not part
+  of our *Early Access* program. Feel free to [apply for early
+  access](https://forms.mergify.com/ci-insights-early-access).
+:::
+
+## Enabling CI Insights for GitHub
+
+1. Enable CI Insights on your repositories by visiting the [GitHub Integrations
+   page](/integrations/github).
```

Comment:
Shouldn't we redirect directly to this page on the dashboard instead ? That's what this line feels like it's going to do when i read it. Modifying the sentence to mention that we mean the documentation page would also work if we can't or don't want to redirect to the dashboard.

Reply:
Good point, I've added both.